setup.py

```diff
@@ -8,9 +8,9 @@
     author='LightStep',
     license='',
     install_requires=['thrift==0.9.2',
-                      'opentracing==1.0rc3',
+                      'opentracing>=2.0.0,<3.0',
```

Comment:
(note that these are not on pypi yet, though the necessary code is in master for opentracing and basictracer)